### PR TITLE
fixed the styling of footer in the Homepage

### PR DESCRIPTION
**Pull request by bipin-shrestha**

### CODE REVIEW

File: src/components/common/ParentFooter.js [hunk 1]
```diff
@@ -37,6 +35,7 @@ export default function ParentFooter(props) {
             </Link>
           </div>
         </nav>
+        <div class="copyright">©{curYear} Story Squad HQ</div>
```
Comment (PVigar88) [inline src/components/common/ParentFooter.js:38]: Nice simple fix to clean up the layout of the footer!

**Comment by bipin-shrestha** [comment]
[Fix the Footer tab in homepage.](https://trello.com/c/zQ37fqOe/764-fix-the-footer-tab-in-homepage)

File: src/styles/less/ParentFooter.less [hunk 2]
```diff
@@ -1,43 +1,41 @@
...
-      color: rgb(102,154,249);
-    }
-  };
\ No newline at end of file
+  }
+
+  a:hover {
+    color: rgb(102, 154, 249);
+  }
+}
```
Comment (Mitch98k) [inline src/styles/less/ParentFooter.less:41]: Is this the closing tag for the footer styling, and if so, why not put it before the inner container styling on line 18? If not, where is the closing tag?

File: src/components/common/ParentFooter.js (hunk 1 above)
Comment (Mitch98k) [inline src/components/common/ParentFooter.js:38]: Great job moving the copyright to under the links.

**Review by Mitch98k — CHANGES_REQUESTED** [review]

File: src/styles/less/ParentFooter.less (hunk 2 above)
Comment (bipin-shrestha) [reply src/styles/less/ParentFooter.less:41]: Yes, it is the closing tag for the footer styling.

**Review by PVigar88 — APPROVED** [review]
Looks good. Should clean up the footer layout nicely

File: src/components/common/ParentFooter.js (hunk 1 above)
Comment (bipin-shrestha) [reply src/components/common/ParentFooter.js:38]: Thanks Paul.

Comment (bipin-shrestha) [reply src/components/common/ParentFooter.js:38]: Thanks Mitchell.

File: src/styles/less/ParentFooter.less (hunk 2 above)
Comment (Mitch98k) [reply src/styles/less/ParentFooter.less:41]: Ok. I'm not sure if it matters, but why is it all the way down there? Is there a reason to include all the other stying inside the footer bracket?

**Review by bipin-shrestha — COMMENTED** [review]
All Resolved

File: src/styles/less/ParentFooter.less (hunk 2 above)
Comment (bipin-shrestha) [reply src/styles/less/ParentFooter.less:41]: Just checked putting the closing tag, wouldn't make any difference. We can use either way I believe.

**Review by bipin-shrestha — COMMENTED** [review]
resolved

File: src/styles/less/ParentFooter.less (hunk 2 above)
Comment (Mitch98k) [reply src/styles/less/ParentFooter.less:41]: Ok then.

**Review by Mitch98k — APPROVED** [review]

**Pull request merged**